js/src/figure.js

```diff
@@ -142,7 +150,8 @@ var FigureView = widgets.DOMWidgetView.extend( {
         this.wire_box.add(make_line(-0.5, -0.5+1, -0.5, -0.5, -0.5+1, -0.5+1, this.axes_material))
         this.wire_box.add(make_line(-0.5+1, -0.5+1, -0.5, -0.5+1, -0.5+1, -0.5+1, this.axes_material))
 
-        this.camera.position.z = 2
+        // set a good intial z for any fov angle
+        this.camera.position.z = 2 * this.getTanDeg(45/2) / this.getTanDeg(this.model.get("camera_fov")/2)
```

Comment:
Ah, by magic numbers I mean any number that appears in code without explanation (e.g. `3.14...`). These should typically be replaced by constant vars, or at least commented.